build/cargo.gni

```diff
@@ -19,7 +19,15 @@ rust_lib_output_path =
 
 template("rust_ffi") {
   static_library(target_name) {
-    libs = [ rust_lib_output_path ]
+    libs = []
+    deps = []
+    if (!is_component_build) {
+      libs += [ "$root_build_dir/obj/base/libbase.a" ]
+      deps += [ "//base" ]
```

Comment:
Can you please explain a bit (or comment) what this is doing? Is it that our rust code needs to link to `base` to get the correct `rust_eh_personality` routine?